content/03-reference/01-tools-and-interfaces/06-prisma-studio.mdx

```diff
@@ -7,7 +7,13 @@ experimental: true
 
 ## Overview
 
-Prisma Studio is a visual editor for your database. To run Prisma Studio, run the following command in your terminal:
+Prisma Studio is a very powerful database GUI where you can interact with your data.
+
+## Not Your Typical Database GUI
+
+Prisma Studio is different from a typical database GUI program (such as [TablePlus](https://tableplus.com/)) in that it provides a layer of abstraction which allows you to see your data represented as it is in your Prisma datamodel. This is one of the several ways that Prisma bridges the gap between how you structure and interact with your data in your application and how it is actually structured and represented in the underlying database. One benefit of this is that it helps you to build intuition and understanding of these two linked but separate layers over time.
```

Comment:
> which allows you to see your data represented as it is in your Prisma datamodel

- "datamodel" should be spelled as two words: "data model" (sorry this is confusing, in Prisma 1 we had the convention to spell it as one word but with Prisma 2 it's two words). Ideally it can also link to the "Data model" docs page: https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema/data-model